packages/react/src/components/navigation/sideBar/Item/Item.tsx

```diff
@@ -95,6 +111,7 @@ const Item = ({ label, children, onClick, isActive, isDisabled }: ItemType): JSX
       >
         <ItemLabel variant="paragraph">{label}</ItemLabel>
       </TransitionInOut>
+      {displayNotificationBadge && <Badge />}
```

Comment:
Wondering if the badge should not be inside the transition scope 🤔

Reply:
I does if we want it to disappear with the Text. I actually did it now, in order to fix the positioning when it's collapsed, I make the one beside the text disappear and another one appear in the right corner of the icon. It's the exact same behaviour we have in the current Ledger Live menu.